lib/view_component/form/helpers/rails_backports.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module Form
+    module Helpers
+      module RailsBackports
+        # Backport field_id from Rails 7.0
+        if ::Rails::VERSION::MAJOR < 7
```

Comment:
Maybe we may rename the module as `Rails7Backports` and move the condition on `Form::Builder` and import this module only on Rails < 7 versions.